test/test_automl/test_estimators.py

```diff
@@ -227,6 +227,26 @@ def test_cv_results(self):
         del cls
         self._tearDown(output)
 
+    def test_f_contiguous_array(self):
+
+        output = os.path.join(self.test_dir, '..', '.tmp_estimator_fit')
+        self._setUp(output)
+
+        X_train, Y_train, X_test, Y_test = putil.get_dataset('iris')
+        X_train = np.asfortranarray(X_train)
+        automl = AutoSklearnClassifier(time_left_for_this_task=15,
+                                       per_run_time_limit=5,
+                                       tmp_folder=output,
+                                       output_folder=output)
+        automl.fit(X_train, Y_train)
```

Comment:
I think it would be good if you extract the name generation into it's own function. Then there is no need to call fit, which saves time when running the unit tests.